test/io.jl

```diff
@@ -130,6 +130,27 @@ end
 
     @test_throws ArgumentError DataFrames._show(stdout, MIME("text/html"),
                                                 DataFrame(ones(2,2)), rowid=10)
+
+    df = DataFrame(
+        A=Int64[1,4,9,16],
+        B = [
+            md"[DataFrames.jl](http://juliadata.github.io/DataFrames.jl)",
+            md"###A",
+            md"``\frac{A}{B}``",
+            md"*A*b**A**" ]
+    )
+
+    @test repr(MIME("text/html"), df) ==
```

Comment:
Can you also use triple-quoted strings here and below?

Reply:
As best I can tell, Julia 1.0.5 has issues with triple-quoted strings.  When I changed some of the test cases to use TQS, Julia 1.0.5 inserts spaces in front of each line.  So on one hand I'm being asked to use TQS and on the second hand I'm being asked to indent by 8 spaces.  

The DataFrames.jl/test/io.jl:26 works on Julia 1 but not Julia 1.0 because Julia 1.0 does not strip spaces before each TQS line. (At least that what it appears....)
```
LaTeX export: Test Failed at /home/travis/build/JuliaData/DataFrames.jl/test/io.jl:26

  Expression: repr(MIME("text/latex"), df) == str

   Evaluated: "\\begin{tabular}{r|ccccccc}\n\t& A & B & C & D & E & F & G\\\\\n\t\\hline\n\t& Int64 & String & String & Float64? & Cat…? & String & MD…\\\\\n\t\\hline\n\t1 & 1 & \\\$10.0 & A & 1.0 & a & \\emph{\\#undef} & \\href{http://juliadata.github.io/DataFrames.jl}{DataFrames.jl}\n\n \\\\\n\t2 & 2 & M\\&F & B & 2.0 & \\emph{missing} & \\emph{\\#undef} & \\#\\#\\#A\n\n \\\\\n\t3 & 3 & A\\textasciitilde{}B & C & \\emph{missing} & c & \\emph{\\#undef} & \$\\frac{A}{B}\$\n\n \\\\\n\t4 & 4 & \\textbackslash{}\\textbackslash{}alpha & S & 3.0 & d & \\emph{\\#undef} & \\emph{A}b\\textbf{A}\n\n \\\\\n\\end{tabular}\n" == "        \\begin{tabular}{r|ccccccc}\n        \t& A & B & C & D & E & F & G\\\\\n        \t\\hline\n\t& Int64 & String & String & Float64? & Cat…? & String & MD…\\\\\n        \t\\hline\n        \t1 & 1 & \\\$10.0 & A & 1.0 & a & \\emph{\\#undef} & \\href{http://juliadata.github.io/DataFrames.jl}{DataFrames.jl}\n\n \\\\\n        \t2 & 2 & M\\&F & B & 2.0 & \\emph{missing} & \\emph{\\#undef} & \\#\\#\\#A\n\n \\\\\n        \t3 & 3 & A\\textasciitilde{}B & C & \\emph{missing} & c & \\emph{\\#undef} & \$\\frac{A}{B}\$\n\n \\\\\n        \t4 & 4 & \\textbackslash{}\\textbackslash{}alpha & S & 3.0 & d & \\emph{\\#undef} & \\emph{A}b\\textbf{A}\n\n \\\\\n        \\end{tabular}\n"
```

Reply:
Second, triple quote strings are not ideal when the text represents a single line of text (not a multi-line block).  HTML output is formatted as a single line.

Reply:
I find this surprising as I never encountered this problem. And the existing tests passed on Julia 1.0... But I know it can be hard to get spacing right with triple quotes. I think you need to put the closing `"""` back on its own line (it makes a difference).

It's OK to keep HTML as plain strings as indeed there aren't any line breaks there.

Reply:
Handling of triple quoted strings changed in 1.5 https://github.com/JuliaLang/julia/blob/v1.5.0/NEWS.md#language-changes (but I have not checked if this affects this case - probably not)

Reply:
@bkamins I think it is the issue you point to with the handling of escaped characters in  <1.5 that is giving me the grief.

Reply:
I see. Then use actual empty lines in triple-quoted strings instead of `\n`. It's more logical anyway.

Reply:
It's `\t` that are the problem.

Reply:
Not AFAICT. There are already uses of `\t` in the existing test. What's new is the use of `\n`. In my tests the following works on Julia 1.0:
```julia

str = """
    \\begin{tabular}{r|ccccccc}
    \t& A & B & C & D & E & F & G\\\\
    \t\\hline
    \t& Int64 & String & String & Float64? & Cat…? & String & MD…\\\\
    \t\\hline
    \t1 & 1 & \\\$10.0 & A & 1.0 & a & \\emph{\\#undef} & \\href{http://juliadata.github.io/DataFrames.jl}{DataFrames.jl}

     \\\\
    \t2 & 2 & M\\&F & B & 2.0 & \\emph{missing} & \\emph{\\#undef} & \\#\\#\\#A

     \\\\
    \t3 & 3 & A\\textasciitilde{}B & C & \\emph{missing} & c & \\emph{\\#undef} & \$\\frac{A}{B}\$

     \\\\
    \t4 & 4 & \\textbackslash{}\\textbackslash{}alpha & S & 3.0 & d & \\emph{\\#undef} & \\emph{A}b\\textbf{A}

     \\\\
    \\end{tabular}
    """
```

Reply:
You are correct.